File: app/src/main/java/com/duy/interpreter/exceptions/grouping/GroupingException.java
package com.duy.interpreter.exceptions.grouping;

import com.duy.interpreter.exceptions.ParsingException;
import com.duy.interpreter.linenumber.LineInfo;


public class GroupingException extends ParsingException {

    public GroupingException(LineInfo line, String message) {
        super(line, message);
    }

    public GroupingException(LineInfo line) {
        super(line);
    }
}

File: libCompiler/src/main/java/com/duy/pascal/backend/parse_exception/grouping/GroupingException.kt
package com.duy.pascal.backend.parse_exception.grouping

import com.duy.pascal.backend.parse_exception.ParsingException
import com.duy.pascal.backend.linenumber.LineInfo


open class GroupingException : ParsingException {

    constructor(line: LineInfo, message: String) : super(line, message) {}

    constructor(line: LineInfo) : super(line) {}
}
